static/locales/en.json

```
@@ -1577,7 +1579,7 @@
             "vllm_omni": "vLLM-Omni (Local Clone)"
         },
         "sourcePreset": "Preset",
-        "sourceClone": "Clone",
+        "sourceClone": "Voice Cloning",
```

Biome (2.5.1)

[error] The key sourceClone was already declared.

(lint/suspicious/noDuplicateObjectKeys)